sepal_ui/scripts/gee.py

```diff
@@ -133,3 +134,44 @@ def is_asset(asset_name: str, folder: Union[str, Path] = "") -> bool:
             break
 
     return exist
+
+
+@sd.need_ee
+def delete_assets(asset_id: str) -> None:
+    """Delete the selected asset and all its content.
+
+    Args:
+        asset_id: the Id of the asset
```

Comment:
Can we add here something like: `the id of the asset or a gee folder"?

Reply:
I'll add it but be careful it's misguiding, folders for GEE are assets as well, assets of type "FOLDER". That's actually why `get_assets` is a bit convoluted.